specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/definitions.json

```diff
@@ -2830,6 +2869,53 @@
       },
       "description": "Paged OpenIdProviders list representation."
     },
+    "OpenidConnectProviderCreateContract": {
```

Comment:
>OpenidConnectProviderCreateContract [](start = 5, length = 35)

maybe we can have `OpenIDConnectProvider` 
[https://auth0.com/docs/protocols/oidc](https://auth0.com/docs/protocols/oidc)